migrate.py

```diff
@@ -205,7 +205,9 @@ def visit_Assign(self, node):
     deps = []
     for fragment in doc_finder.fragments:
         try:
-            fragment_collection = get_plugin_collection(fragment, 'doc_fragments', spec)
+            # some doc_fragments use subsections (e.g. vmware.vcenter_documentation)
+            fragment_name = fragment.split('.')[0]
+            fragment_collection = get_plugin_collection(fragment_name, 'doc_fragments', spec)
```

Comment:
try/except should only wrap this line since this is where we expect the exception to pop up